packages/app-check/src/index.ts

```diff
@@ -94,6 +99,8 @@ initializeDebugMode();
 declare module '@firebase/app-types' {
   interface FirebaseNamespace {
     appCheck(app?: FirebaseApp): FirebaseAppCheck;
+    ReCaptchaV3Provider: typeof ReCaptchaV3Provider;
+    CustomProvider: typeof CustomProvider;
```

Comment:
Use the classes exported from the types package, otherwise people may get type errors.

Reply:
Done, also made casing consistent.